nengo/networks/workingmemory.py

```diff
@@ -31,22 +30,20 @@ def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
         nengo.Connection(net.mem.output, net.diff.input, transform=-1)
 
         # feed difference into integrator
-        with mem_config:
-            nengo.Connection(net.diff.output, net.mem.input,
-                             transform=difference_gain)
+        nengo.Connection(net.diff.output, net.mem.input,
+                         transform=difference_gain,
+                         synapse=nengo.Lowpass(difference_synapse))
```

Comment:
> But, it should be noted that you could instead pass in a different Synapse class for difference_synapse etc, hence why we don't want to unconditionally make it a Lowpass.

Exactly.  Basically, setting a synapse to a floating point value is a shortcut for one particular type of synapse (Lowpass), but there's no reason to hard-code this to only allow that sort of Synapse.